client/src/components/common/campaignCard/index.jsx

```diff
@@ -10,21 +12,26 @@ const { Text } = Typography;
 const { Meta } = Card;
 
 function Campaign({
+  id,
   title,
   description,
   imgSrc,
   categoryIcon,
 }) {
+  const navigate = useNavigate();
   return (
     <Card
+      onClick={() => navigate(`/campaign/${id}`)}
       className="customCard"
       hoverable
       cover={<img alt="card cover" src={imgSrc} />}
     >
       <img className="category" alt="Category" src={categoryIcon} />
       <Meta className={styles.title} title={title} />
       <Text type="secondary">{description}</Text>
-      <Button type="primary">
+      <Button
+        type="primary"
+      >
         Donate
```

Comment:
I want to change this to `Read more` and when clicked it goes to campaign single page